src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -1129,21 +1112,13 @@ public virtual bool ReadToNextSibling(string localName, string namespaceURI)
         // Returns true if the given argument is a valid Name.
         public static bool IsName(string str)
         {
-            if (str == null)
-            {
-                throw new NullReferenceException();
-            }
-            return ValidateNames.IsNameNoNamespaces(str);
+            return str == null ? throw new NullReferenceException() : ValidateNames.IsNameNoNamespaces(str);
```

Comment:
I think these will be fine to replace with ArgumentNullException

Reply:
@krwq done